src/Tokenizer/Tokens.php

```diff
@@ -1083,6 +1083,13 @@ public function isMonolithicPhp()
             ) {
                 return false;
             }
+
+            if ($this[$index]->isGivenKind(T_CLOSE_TAG)) {
```

Comment:
@keradus besides all the descriptions, I would like you to take a look at this fix :)